Tests/EntitlementsSpec.swift

```diff
@@ -0,0 +1,394 @@
+                    expect(flags).to(beEmpty())
+                }
+                
+                it("returns feature flags dictionary when claim exists") {
+                    let flags = featureFlagsService.getFeatureFlags()
+                    expect(flags).to(beAKindOf([String: Any].self))
+                }
+            }
+            
+            describe("isFeatureEnabled") {
+                it("returns false for non-existent feature flag") {
+                    let result = featureFlagsService.isFeatureEnabled(code: "non_existent_flag")
+                    expect(result).to(equal(false))
+                }
+                
+                it("returns default value for non-existent feature flag") {
+                    let result = featureFlagsService.isFeatureEnabled(code: "non_existent_flag", defaultValue: true)
+                    expect(result).to(equal(true))
+                }
+            }
+            
+            describe("getFeatureFlag") {
+                it("returns nil for non-existent feature flag") {
+                    let result = featureFlagsService.getFeatureFlag(code: "non_existent_flag")
+                    expect(result).to(beNil())
+                }
+                
+                it("returns Any? type for feature flag") {
+                    let result = featureFlagsService.getFeatureFlag(code: "test_flag")
+                    // Result will be nil when flag doesn't exist
+                    expect(result).to(beNil())
+                }
+            }
+        }
+        
+        describe("ClaimsService") {
+            var auth: Auth!
+            var claimsService: ClaimsService!
+            
+            beforeEach {
+                KindeSDKAPI.configure()
+                auth = KindeSDKAPI.auth
+                claimsService = auth.claims
+            }
+            
+            describe("getClaim") {
+                it("returns nil for non-existent claim") {
+                    let result = claimsService.getClaim(forKey: "non_existent_claim")
+                    expect(result).to(beNil())
+                }
+                
+                it("returns claim value when claim exists") {
+                    // This would require a mock token with specific claims
+                    // TODO: Add proper test fixtures with mocked tokens
+                    pending("Requires mock token with claims") {}
+                }
+            }
+            
+            describe("getClaimValue") {
+                it("returns nil for non-existent claim value") {
+                    let result = claimsService.getClaimValue(forKey: "non_existent_claim")
+                    expect(result).to(beNil())
+                }
+                
+                it("returns claim value when claim exists") {
+                    let result = claimsService.getClaimValue(forKey: "test_claim")
+                    expect(result).to(beAKindOf(Any.self))
```

Comment:
This tests doesn't pass as it requires the mock token just like its similar test for  the `getClaim` method.

We can add this pending note here.

```suggestion
                    // This would require a mock token with specific claims
                    // TODO: Add proper test fixtures with mocked tokens
                    pending("Requires mock token with claims") {}
```